jbpm-flow/src/main/java/org/jbpm/process/core/timer/impl/QuartzNameHelper.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jbpm.process.core.timer.impl;
+
+import org.drools.core.time.JobContext;
+import org.jbpm.process.core.timer.NamedJobContext;
+import org.jbpm.process.instance.timer.TimerManager.ProcessJobContext;
+import org.jbpm.process.instance.timer.TimerManager.StartProcessJobContext;
+import org.kie.api.runtime.EnvironmentName;
+
+class QuartzNameHelper {
```

Comment:
Shouldn't it be "public"?

Reply:
Once this class is used also for EJbTimer, yes, it should be public, before was package becuase it was only used for Quartz ones